pages/signup.php

```diff
@@ -79,58 +75,56 @@
 {
   width:auto;
 }
-
+/* On screens that are 992px or less, set the background color to blue */
+@media screen and (max-width: 992px) {
+  #showpass
+  {
+    width:38vw;
+  }
+}
 </style>
 
-<?php include("../templates/header.php"); ?>
+<?php include "../templates/header.php";?>
 
 	<div class="nav-container">
-      <?php include("../templates/navBar.php") ?>
+      <?php include "../templates/navBar.php"?>
   </div>
 
+  <div class="container col-lg-5 mt-0 co-sm-11">
 	<form action="../include/signup.inc.php" method="POST" id=signupForm>
 
-    <div class="form-elements">
+    <div class="form-elements pt-4">
       <h3>SIGN UP</h3>
     </div>
 
     <div class="form-elements">
         <img src="../images/red-avatar.png" alt="Avatar" class="avatar">
-        <?php 
```

Comment:
Hey, I am not sure if the formatting is done properly. This whole section seems to be deleted and added back. This might be due to the `spaces` size that you use in your `vscode`. It must be set to `2`, I think.

Reply:
Also, there are many changes done in this PR so, it is hard to test them. Can you break this PR into 2-3 different PRs?
I will added `Level 3` for all of them.

Reply:
Ok i am making a fresh PR only for responsiveness of login and sign up page..

@siv2r  Is that ok?

Reply:
make separate pr for both logins and sign up. Also, you can get more points this way ;)

Reply:
Ok